.github/workflows/build.yaml

```diff
@@ -201,6 +201,46 @@ jobs:
           name: ${{matrix.platform}}-lr-logs
           path: '*-logs.tar.gz'
 
+  libtester-deb-install-test:
+    name: libtester deb install test
+    needs: [d, Build, dev-package]
+    if: always() && needs.dev-package.result == 'success'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu20] # Assuming want to do both ubuntu20 and ubuntu22 at some point so setting up with matrix
+    runs-on:  ["self-hosted", "enf-x86-lowtier"] # not sure if this warrants a better machine, but start here
+    container: ${{fromJSON(needs.d.outputs.p)['ubuntu20'].image}}
```

Comment:
Since you're using the cached based image, this test won't really check that leap-dev.deb's dependency packages , which are _manually_ set here and thus prone to getting messed up,
https://github.com/AntelopeIO/leap/blob/651d21f2a3a988a019ff58745d125464c310c201/package.cmake#L64
are correct/exhaustive because the cached base image already installed those for building leap itself.

**This is probably an okay limitation for now though.**

But if you're curious to explore a solution, I think you could resolve this by just using ubuntu's base image -- `container: ubuntu:focal` or such, but this comes at the cost of being more difficult to use the matrix set up you have for futureproofing. My hunch is you might be able to do something like this since you'd only have 20 & 22:
```yaml
matrix:
  platform: [ubuntu20, ubuntu22]
container: ${{matrix.platform == 'ubuntu20' && 'ubuntu:focal' || 'ubuntu:jammy'}}

name: leap-dev-${{matrix.platform}-amd64
```

 (you would also need to leave cmake install here too https://github.com/AntelopeIO/leap/pull/1259/files#r1224554620)

Reply:
I honestly had this thought, but wasn't sure if we were ready to move toward the base container images and build up from those for this task or if that was planned for a later effort.  With wanting to get something up and testing/working, I chose to start with the cached images knowing it wouldn't be too difficult to then pivot toward the other.

Reply:
Addressed.